app/plugin_install.go

```diff
@@ -107,6 +123,27 @@ func (a *App) RemovePlugin(id string) *model.AppError {
 }
 
 func (a *App) removePlugin(id string) *model.AppError {
+	if rmErr := a.removePluginLocally(id); rmErr != nil {
+		return rmErr
+	}
+
+	// Remove bundle from the file store.
+	storePluginFileName := filepath.Join("./plugins", id) + ".tar.gz"
```

Comment:
Since this appears twice, suggest we factor out as a helper function.

Reply:
Done